src/components/application_manager/src/request_controller.cc

```diff
@@ -81,6 +81,7 @@ void RequestController::InitializeThreadpool() {
   pool_state_ = TPoolState::STARTED;
   char name[50];
   for (uint32_t i = 0; i < pool_size_; i++) {
+    // cppcheck-suppress invalidPrintfArgType_sint
     snprintf(name, sizeof(name) / sizeof(name[0]), "AM Pool %d", i);
```

Comment:
@Ypostolov Do not suppress this issue. The solution you can find at several comment above.